0000-template.md

```diff
@@ -0,0 +1,101 @@
+# Title (Ex. 0000: RFC Topic)
```

Comment:
I note that the Rust RFCs have moved well into the 5 digit range, so I suggest we plan ahead and start with the template as #00000.

We can only hope...

Reply:
:+1: